confiture-web-app/src/components/audit/NotesModal.vue

```diff
@@ -19,97 +31,199 @@ defineExpose({
   hide: () => modal.value?.hide()
 });
 
+const showFileSizeError = ref(false);
+const showFileFormatError = ref(false);
+
 const auditStore = useAuditStore();
+const notify = useNotifications();
+const route = useRoute();
 
 const modal = ref<InstanceType<typeof DsfrModal>>();
 const isOffline = useIsOffline();
 
-const notes = ref("");
+const notes = ref(auditStore.currentAudit?.notes || "");
 
-function handleSubmit() {
-  emit("confirm", notes.value);
-}
+const uniqueId = computed(() => route.params.uniqueId as string);
+const files = computed(() => auditStore.currentAudit?.notesFiles || []);
 
-function handleClose() {
-  modal.value?.hide();
+const handleNotesChange = debounce(() => emit("confirm", notes.value), 500);
+
+function handleUploadFile(file: File) {
+  showFileSizeError.value = false;
+  showFileFormatError.value = false;
+  notify("info", "Chargement en cours");
+  auditStore
+    .uploadAuditFile(uniqueId.value, file)
+    .then(() => {
+      notify("success", "Fichier téléchargé avec succès.");
```

Comment:
Pareil pour les toasts :

```suggestion
      notify("success", undefined, "Fichier téléchargé avec succès.");
```